src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/CryptoConfig.cs

```diff
@@ -291,12 +329,19 @@ public static object CreateFromName(string name, params object[] args)
                 throw new ArgumentNullException(nameof(name));
 
             Type retvalType = null;
+            
+            // Check to see if we have an applicaiton defined mapping
+            lock (InternalSyncObject) {
```

Comment:
Style: mscorlib (netfx) used K&R bracing (same line for opening brace), but in corefx we use BSD bracing (opening brace on new line indented to the statement that it belongs to).  Please convert the bracing style of this new code to match what's already prevalent in this file.